hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/flink/FlinkCleaner.java

```diff
@@ -56,6 +56,12 @@ public void deleteOrphanJobs(Timer timer) {
     //Get all jobs from history server
     DistributedFileSystemOps dfso = null;
     try {
+      if(!flinkController.isFlinkInstalled()) {
```

Comment:
@robzor92 @SirOibaf isn't it better to get this information from HostServices?

Reply:
That's probably true, sound ok @SirOibaf ?

Reply:
I think this solution is slightly better. Here you also cover the case in which the history server has been deployed, but Hopsworks for whatever reason doesn't have access to the configuration file

Reply:
@SirOibaf that reason is an exceptional case, as in an exception would be thrown  because if the service is installed then the directory must be there and if the exception is caught then we can cancel the timer. We wouldn't have defensive code for non-exceptional cases.

Reply:
I updated the code, have a look and see what you think. If it is not listed as a service then obviously it should be cancelled. Otherwise any exception happening during exception is logged. I don't think we should add more checks that would cancel the timer, feels like we are over-engineering this.